airbyte-scripts/github-filler/requirements.txt

```diff
@@ -0,0 +1,2 @@
+requests==2.26.0
+
```

Comment:
i dont sure if we need this file, the requests is the part of standard library, or latest / specificversion is required?